packages/iris-grid/src/sidebar/WritableStream.ts

```diff
@@ -0,0 +1,6 @@
+import { WritableStream as ponyfillWritableStream } from 'web-streams-polyfill/ponyfill';
+
+// WritableStream is not supported in Firefox (also IE) yet. use ponyfillWritableStream instead
```

Comment:
Support was added in firefox 100, browserlist `defaults` has 102 as the minimum supported version. So we should actually be able to drop this polyfill entirely now, no?

![image](https://user-images.githubusercontent.com/1576283/228648946-e2362db0-1cf8-4056-984c-39c547483d2f.png)

![image](https://user-images.githubusercontent.com/1576283/228649174-cdc022f7-fec6-4de5-9fa6-38c5095628a6.png)

Reply:
Yee